packages/amplify-provider-awscloudformation/src/root-stack-builder/root-stack-builder.ts

```diff
@@ -0,0 +1,252 @@
+import * as cdk from '@aws-cdk/core';
```

Comment:
Could you split this into 3 files and so we can keep one class per file. For importing, you can create and `index.ts` and re-export it there